ramls/schema/authority.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Add missing properties for fixed and variable tags

Reply:
Yes, it is done